doc/postprocess.py

```diff
@@ -26,9 +26,7 @@ def _open(fn, *args, **kwargs):
         r"""Handle UTF-8 encoding when loading under Py3"""
         # Issue is that default encoding under Py3 is
         # locale dependent (might even be ASCII),
-        # so need to specify the encoding.  Py2 doesn't care.
-        if sys.version_info.major < 3:
-            return open(fn, *args, **kwargs)
+        # so need to specify the encoding.
```

Comment:
according the the [3.8 documentation](https://docs.python.org/3.8/library/io.html) `io.open` is an alias for `open`. Is there a reason why `open` can't be used here? 
On Py3 specifying the encoding means that the file `mode` can't contain `b`, is there a possibility that might happen?.

Reply:
The only two uses are on the lines below (one with `mode='r'` and one with `mode='w'`). I've inlined the function with the comment at the top.